src/runtime/server/api/highlight.ts

```diff
@@ -87,7 +87,14 @@ export default defineLazyEventHandler(async () => {
 
     // Load supported language on-demand
     if (!highlighter.getLoadedLanguages().includes(lang)) {
-      await highlighter.loadLanguage(lang)
+      // eslint-disable-next-line no-console
+      console.warn(`[Nuxt] [Content] Language "${lang}" is not loaded Shiki. Falling back to plain code.`)
```

Comment:
Cannot we use our logger here instead? https://github.com/nuxt/content/blob/main/src/utils.ts#L11

```suggestion
      console.warn(`Language "${lang}" is not loaded Shiki. Falling back to plain code.`)
```